scripts/microannotate_generator.py

```diff
@@ -29,37 +30,20 @@ def __init__(self, cache_root, repo_url, tokenize, remove_comments):
         self.repo_dir = os.path.join(cache_root, "mozilla-central")
 
     def generate(self):
-        repository.clone(self.repo_dir)
 
-        logger.info("mozilla-central cloned")
+        with concurrent.futures.ThreadPoolExecutor(max_workers=2) as executor:
+            executor.submit(repository.clone, self.repo_dir)
+            logger.info("Cloning mozilla-central")
```

Comment:
You can keep the "mozilla-central cloned" log at the end of cloning, by using https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Future.add_done_callback on the future returned by executor.submit

Reply:
Thanks, I have updated PR.
Can you please re review ?